lib/wx/configure.in

```diff
@@ -28,6 +28,20 @@ AC_PREREQ(2.59)
 if test -f ./CONF_INFO; then
    rm ./CONF_INFO
 fi
+if test -f ./SKIP; then
```

Comment:
Maybe we should remove this,   touch SKIP files was a way to skip building before configure was fixed, and might still be used.

Reply:
Yeah, wasn't sure on this one. Had added after running ./configure twice and realizing that it wasn't cleaned up.